Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ContextMenu.cpp

```diff
@@ -22,60 +23,35 @@
 #include <EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/BlendGraphWidget.h>
 #include <EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NodePaletteWidget.h>
 #include <EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NodePaletteModelUpdater.h>
+#include <EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/SolidColorIconEngine.h>
 #include <EMotionStudio/Plugins/StandardPlugins/Source/MotionSetsWindow/MotionSetsWindowPlugin.h>
 #include <GraphCanvas/Widgets/NodePalette/TreeItems/NodePaletteTreeItem.h>
 #include <GraphCanvas/Widgets/NodePalette/NodePaletteWidget.h>
+#include <QItemSelectionModel>
 #include <QMenu>
 #include <QWidgetAction>
 
-
 namespace EMStudio
 {
-    void BlendGraphWidget::AddNodeGroupSubmenu(QMenu* menu, EMotionFX::AnimGraph* animGraph, const AZStd::vector<EMotionFX::AnimGraphNode*>& selectedNodes)
+    void BlendGraphWidget::AddAssignNodeToGroupSubmenu(QMenu* menu, EMotionFX::AnimGraph* animGraph)
     {
-        // node group sub menu
-        QMenu* nodeGroupMenu = new QMenu("Node Group", menu);
-        bool isNodeInNoneGroup = true;
-        QAction* noneNodeGroupAction = nodeGroupMenu->addAction("None");
-        noneNodeGroupAction->setData(qulonglong(0)); // this index is there to know it's the real none action in case one node group is also called like that
-        connect(noneNodeGroupAction, &QAction::triggered, this, &BlendGraphWidget::OnNodeGroupSelected);
-        noneNodeGroupAction->setCheckable(true);
-
         const size_t numNodeGroups = animGraph->GetNumNodeGroups();
+        if (numNodeGroups == 0)
+        {
+            return;
+        }
+
+        QMenu* nodeGroupMenu = new QMenu("Assign To Node Group", menu);
```

Comment:
```suggestion
        QMenu* nodeGroupMenu = new QMenu(tr("Assign To Node Group"), menu);
```

Reply:
Whoops, good catch. I now notice the strings in the rest of the file aren't wrapped in `tr` either. I can fix that.